odoo/addons/test_new_api/models/test_new_api.py

```diff
@@ -97,6 +97,7 @@ class Discussion(models.Model):
     important_emails = fields.One2many('test_new_api.emailmessage', 'discussion',
                                        domain=[('important', '=', True)])
 
+    history = fields.Json('History', default={'delete_messages': []})
```

Comment:
Oooh, I never saw anything related to `fields.Json` yet! Is there any info/docs about it? 🤔 
CC @mart-e :D

Reply:
@Yenthe666 
Hello, this `fields.Json` is mainly implemented for analytics. It will not get most functions (like search), atleast for now.
There will be infos in the doc not to consider this field as usable as the others.

Reply:
@gawa-odoo ah ok, I see :) I was excited for a `Json` field to use on the fields level for API data storage haha :) Thanks for the feedback!